client/src/app/core/core-services/operator.service.ts

```diff
@@ -415,6 +417,49 @@ export class OperatorService {
         return result;
     }
 
+    /**
+     * Checks, if the own OML is equals or higher than at least one of the given permissions.
+     *
+     * @param permissionsToCheck The required permissions to check.
+     *
+     * @returns A boolean whether an operator's OML is high enough.
+     */
+    public hasOrganizationPermissions(...permissionsToCheck: OML[]): boolean {
+        return permissionsToCheck.some(permission => (omlNameMapping[this.OML] || 0) >= omlNameMapping[permission]);
+    }
+
+    /**
+     * Checks, if the own CML is equals or higher than at least one of the given permissions.
+     * If an operator has the permission `OML.can_manage_organisation` or higher, then it will always return `true`.
+     *
+     * @param committeeId The committee's id to know for which committee the CML is checked.
+     * @param permissionsToCheck The required permissions to check.
+     *
+     * @returns A boolean whether an operator's CML is high enough.
+     */
+    public hasCommitteePermissions(committeeId: Id | undefined, ...permissionsToCheck: CML[]): boolean {
+        // If a user can manage an entire organisation, he can also manage every committee.
+        if (!this.CML || !committeeId) {
+            return false;
+        }
+        if (this.hasOrganizationPermissions(OML.superadmin, OML.can_manage_organisation)) {
+            return true;
+        }
+        const currentCommitteePermission = cmlNameMapping[this.CML[committeeId]] || 0;
+        return permissionsToCheck.some(permission => currentCommitteePermission >= cmlNameMapping[permission]);
+    }
+
+    /**
+     * Determines whether the current operator is included in at least one of the committees, which are passed.
+     *
+     * @param committees Several committees to check if the current operator is included in them.
+     *
+     * @returns `true`, if the current operator is included in at least one of the given committees.
+     */
+    public isInCommittees(...committees: Committee[]): boolean {
+        return committees.some(committee => this.userSubject.value.committee_ids?.includes(committee.id));
```

Comment:
If you are using `committee_ids`, you must make sure, the operator requests this field.